2unix

```diff
@@ -30,14 +30,31 @@ source "${NVOC}/1bash"
 current_1bash_ver="$nvOC_1bash_ver"
 source "${NVOC}/1bash.template"
 template_1bash_ver="$nvOC_1bash_ver"
+last_compatible_1bash_ver="$nvOC_1bash_last_compatible"
+last_incompatible_1bash_ver="$nvOC_1bash_last_incompatible"
 
-if [[ ${current_1bash_ver} != ${template_1bash_ver} ]]
+
+if [[ ${current_1bash_ver} -lt ${last_incompatible_1bash_ver} ]]
+then
+  echo -e " $(tput bold)################################################################################ $(tput sgr0)"
+  echo ""
+  echo -e " \e[31m     Your 1bash revision is not compatible with latest update"
+  echo "      Please update your 1bash with latest 1bash.template and restart nvOC"
+  echo "      SSH access will be enabled and this program restarts in a minute"
+  echo -e " $(tput sgr0)"
+  echo -e " $(tput bold)################################################################################ $(tput sgr0)"
+  sudo cp /etc/init/ssh.conf-on /etc/init/ssh.conf
+  sleep 60
+  exit
+fi
+
+if [[ ${current_1bash_ver} le ${last_compatible_1bash_ver} ]]
 then
   echo "################################################################################"
-  echo " !!  Your 1bash settings file revision is different from the template one."
-  echo "     This could potentially cause errors. SSH access is now being enabled to"
-  echo "     let you debug. Please, update your 1bash to comply with the template."
-  echo "     nvOC will now try to run anyway in 10 seconds from now..."
+  echo " !!  Your 1bash revision is older than last compatible version"
+  echo "     There are new features available if you update your 1bash with latest 1bash.template."
```

Comment:
Ok almost fine just replace this line with:

> This could potentially cause errors if you're using features that have been reworked. Updating your 1bash with latest 1bash.template is strongly recommanded.

Then add another if block after this one, condition 1bash rev < 1bash.template rev and use a friendyl informational message like that one you just replaced:

> There are new features available if you update your 1bash with latest 1bash.template.

Avoid enabling ssh for debug in this last case, don't insert any sleep.

Finally a simple note about language: the two revision variable names should be:
least_compatible: lowest possible compatible revision
last_compatible: last of old revisions which is 100% compatible